backend/Dockerfile

```
@@ -11,21 +10,23 @@ WORKDIR /home/app
-RUN --mount=type=secret,id=GH_TOKEN,env=GH_TOKEN npm ci --ignore-scripts
+# Ensure shell script is Linux-compatible even when checked out with CRLF on Windows.
+RUN sed -i 's/\r$//' /home/app/migrate.sh && chmod +x /home/app/migrate.sh
+
+RUN npm ci --ignore-scripts
 
 RUN npx prisma generate
-RUN npx prisma generate --schema=prismaDWH/schema.dwh.prisma
 
 RUN npm run build
 
 RUN mkdir -p dist/src/modules/bpmn && \
```

Hadolint (2.14.0)

[error] This is a unicode space. Delete and retype it.

(SC1018)